source/protocol/generated/lsp-messages-requests.ads

```diff
@@ -53,6 +53,11 @@ package LSP.Messages.Requests is
       params : TextDocumentPositionParams;
    end record;
 
+   type Type_Definition_Request is new RequestMessage with
```

Comment:
Currently files in `generated` directory are generated by a python scripts. See `scripts/generate.py`.

Can you fix the python scripts and do `make generate`?

Reply:
Can you also put a :white_check_mark: in README.md in the table on `typeDefinition` row?

Reply:
Done!